### PR TITLE
added draft of the august 2017 newsletter

### PR DESCRIPTION
**Pull request by kelly-o**

### CODE REVIEW

File: docs/blog/newsletter-august-2017.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+##################################
+Write the Docs Newsletter – August
+##################################
+
+Hi folks!
+
+This month's issue is coming at you from the sunny Bohemian countryside. I'm in the Czech Republic this week visiting family, but next month, I – and several hundred of you – will back for Write the Docs EU 2017! Things are getting very real, in conference-planning land, and you can check out what's on the docket for the main stage in `our newly published talk schedule <http://www.writethedocs.org/conf/eu/2017/news/announcing-schedule/>`_!
```
Comment (thatdocslady) [inline docs/blog/newsletter-august-2017.rst]: EU > Prague (we changed it from region to city)

File: docs/blog/newsletter-august-2017.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+In other news, Slack's been humming away as usual this month, with so many excellent conversations that it's getting harder and harder to choose just a few to cover here. We've also got nine different local meetups happening around the world this month. So even if you can't join us for the conferences, don't miss out on your other opportunities to interact with the community!
+
+Oh, and also, as I mentioned last month, we're looking to expand our little editorial team for the newsletter. Special thanks to Claire Lundeby (@claire.lundeby), who'll be joining the crew this month! We're still looking for one or two more folks who already spend a lot of time on the Write the Docs Slack, and want to help us turn insightful Slack conversations into mini-features. Drop me a message on Slack @kobrien042, if that sounds like you!
+
+In the meantime, let's see what people were talking about this month!
+
+************************************************
+What resume advice is the *right* resume advice?
```
Comment (thatdocslady) [inline docs/blog/newsletter-august-2017.rst:16]: This title is sentence-case and the others are title-case, might be useful to decide on a consistent case (either one is fine by me :)

File: docs/blog/newsletter-august-2017.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* Don't limit your resume to your paying work. Include personal and community projects – stuff you work on in your free time or as a side hustle. Just because you're not getting paid, doesn't mean you're not gaining experience. (XP > GP)
+* Make sure you've got your supporting materials in hand. Craft a tailored cover letter for every job. (No form letters!) Get your LinkedIn profile in shape. Spruce up your portfolio; even if you're just starting out, write up some samples for your favorite tool or process.
+* Apply to the right jobs. Job hunting takes a HUGE amount of effort. Better to invest that time into perfecting a few really exciting applications, than putting minimum effort into dozens of applications you don't really care about.
+
+Jobs and careers are a pretty common theme on the WtD Slack, so we've set up a new channel – #career-advice – to better fill that need. Come hit us up with all your job-hunting, or general career advancement questions!
```
Comment (thatdocslady) [inline docs/blog/newsletter-august-2017.rst]: WtD > WTD (brand thing, plz search and replace everywhere in this doc, yes I know I'm late on the brand guide but it's coming I promise!!)

File: docs/blog/newsletter-august-2017.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+* Treat your resume like any other kind of documentation – start by asking who your reader is, and what they need from your resume. Align your content to the themes and priorities present in their job listing/any other company materials you can get your hands on. Try a `keyword density tool <http://tools.seobook.com/general/keyword-density/)>`_ for help identifying those themes.
+* Don't limit your resume to your paying work. Include personal and community projects – stuff you work on in your free time or as a side hustle. Just because you're not getting paid, doesn't mean you're not gaining experience. (XP > GP)
+* Make sure you've got your supporting materials in hand. Craft a tailored cover letter for every job. (No form letters!) Get your LinkedIn profile in shape. Spruce up your portfolio; even if you're just starting out, write up some samples for your favorite tool or process.
+* Apply to the right jobs. Job hunting takes a HUGE amount of effort. Better to invest that time into perfecting a few really exciting applications, than putting minimum effort into dozens of applications you don't really care about.
```
Comment (thatdocslady) [inline docs/blog/newsletter-august-2017.rst]: Is it "apply to" or "apply for"? Something looks strange but I don't trust myself totally on this one...

File: docs/blog/newsletter-august-2017.rst [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+**********************************
+READMEs and Doc-Driven Development
+**********************************
+Poorly written documentation is pervasive among GitHub repos. Past and upcoming WtD conference talks and earlier Slack threads bear witness to our community's recognition of the problem and our efforts to address it. But it continues to be difficult for developers to jump over to the documentarian role, especially after development gets rolling.
+
+One particpant suggested this `README maturity model<https://github.com/LappleApple/feedmereadmes/blob/master/README-maturity-model.md>`_ to help guide documentation efforts. Others mentioned side-stepping the problem altogether with `documentation-driven developement.<http://blog.izs.me/post/161633971373/documentation-driven-development>`_. Even if you've never heard of it, you can probably guess what it means: you write the documentation before you write the code. Writing comments and the README in advance forces you think through what the code should do and helps you see your product from the user's perspective. It's like test-driven development, but with docs!
```
Comment (thatdocslady) [inline docs/blog/newsletter-august-2017.rst]: double space after "suggested this"

File: docs/blog/newsletter-august-2017.rst [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+For that last point, it can be hard to imagine what colors might look like to the various kinds of colorblindness. But this is the internet; there's tools for that! Folks suggested a couple of different options, including `Color Oracle <http://colororacle.org/>`_, a desktop simulator for the different types of colorblindness, and
+`Color Hexa <http://www.colorhexa.com/>`_, which provides a *wealth* of information about any color you look up, including how it appears for various types of colorblindness.
+
+************************
+Making docs maintainable
+************************
+This month the WtD hivemind tackled one of the holy grail questions: how do we establish a truly maintainable docs system? No surprise that we couldn't come up with a definitive set of practices, but folks contributed a wealth of great practical tips:
+* Implement well-structured information architecture
+* Automate as much as possible, especially in areas of rapid change, and make sure that automation scripts are easy to maintain.
```
Comment (thatdocslady) [inline docs/blog/newsletter-august-2017.rst]: this bullet point is the only one that has a period at the end ;)

File: docs/blog/newsletter-august-2017.rst [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+* Set up continuous integration/continuous deployment on doc builds to promote an agile attitude toward doc contributions, fixes
+* Make the process for contributing as simple as possible
+
+As with all our stories, this conversation will continue to evolve on Slack, so if you have a favorite tip for or example of a maintainable docs system, definitely hop on Slack and share it!
+
+*************************
+Upcoming community events
+*************************
+
+**EU CONFERENCE COUNTDOWN: One Month Left!**
```
Comment (thatdocslady) [inline docs/blog/newsletter-august-2017.rst]: EU > Prague

**Review by thatdocslady — COMMENTED** [review]
Looks great! Some very picky nit-picky pickiness comments ;)

File: docs/blog/newsletter-august-2017.rst (hunk 1 above)
Comment (kelly-o) [reply docs/blog/newsletter-august-2017.rst]: gotcha, will update

File: docs/blog/newsletter-august-2017.rst (hunk 2 above)
Comment (kelly-o) [reply docs/blog/newsletter-august-2017.rst:16]: Ah, yep – work habits intruding here. I'll go with sentence.

File: docs/blog/newsletter-august-2017.rst (hunk 4 above)
Comment (kelly-o) [reply docs/blog/newsletter-august-2017.rst]: Either works. I'll go with 'for'.

File: docs/blog/newsletter-august-2017.rst (hunk 3 above)
Comment (kelly-o) [reply docs/blog/newsletter-august-2017.rst]: Sorry, my bad – I can't get this to stick in my head. I never remember which way I'm supposed to be correcting it to.

File: docs/blog/newsletter-august-2017.rst (hunk 5 above)
Comment (kelly-o) [reply docs/blog/newsletter-august-2017.rst]: 👌

File: docs/blog/newsletter-august-2017.rst (hunk 6 above)
Comment (kelly-o) [reply docs/blog/newsletter-august-2017.rst]: good catch

File: docs/blog/newsletter-august-2017.rst (hunk 7 above)
Comment (kelly-o) [reply docs/blog/newsletter-august-2017.rst]: 👍

File: docs/blog/newsletter-august-2017.rst [hunk 8]
```diff
@@ -67,15 +67,15 @@ As with all our stories, this conversation will continue to evolve on Slack, so
 Upcoming community events
```
Comment (hillaryfraley) [inline docs/blog/newsletter-august-2017.rst]: 61 - add "and" maybe?

"...agile attitude toward doc contributions and fixes"

File: docs/blog/newsletter-august-2017.rst [hunk 9]
```diff
@@ -0,0 +1,84 @@
+##################################
+Write the Docs Newsletter – August
```
Comment (ericholscher) [inline docs/blog/newsletter-august-2017.rst]: Should have a year. August is not a unique identifier.

File: docs/blog/newsletter-august-2017.rst [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+##################################
+
+Hi folks!
+
+This month's issue is coming at you from the sunny Bohemian countryside. I'm in the Czech Republic this week visiting family, but next month, I – and several hundred of you – will back for Write the Docs Prague 2017! Things are getting very real, in conference-planning land, and you can check out what's on the docket for the main stage in `our newly published talk schedule <http://www.writethedocs.org/conf/eu/2017/news/announcing-schedule/>`_!
```
Comment (ericholscher) [inline docs/blog/newsletter-august-2017.rst]: "will back for" -> "will be back for"?

File: docs/blog/newsletter-august-2017.rst [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+In other news, Slack's been humming away as usual this month, with so many excellent conversations that it's getting harder and harder to choose just a few to cover here. We've also got nine different local meetups happening around the world this month. So even if you can't join us for the conferences, don't miss out on your other opportunities to interact with the community!
+
+Oh, and also, as I mentioned last month, we're looking to expand our little editorial team for the newsletter. Special thanks to Claire Lundeby (@claire.lundeby), who'll be joining the crew this month! We're still looking for one or two more folks who already spend a lot of time on the Write the Docs Slack, and want to help us turn insightful Slack conversations into mini-features. Drop me a message on Slack @kobrien042, if that sounds like you!
```
Comment (ericholscher) [inline docs/blog/newsletter-august-2017.rst]: Probably don't need the qualifier of "spend a lot of time" -- might make people self select out.

File: docs/blog/newsletter-august-2017.rst [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+
+One particpant suggested this `README maturity model<https://github.com/LappleApple/feedmereadmes/blob/master/README-maturity-model.md>`_ to help guide documentation efforts. Others mentioned side-stepping the problem altogether with `documentation-driven developement.<http://blog.izs.me/post/161633971373/documentation-driven-development>`_. Even if you've never heard of it, you can probably guess what it means: you write the documentation before you write the code. Writing comments and the README in advance forces you think through what the code should do and helps you see your product from the user's perspective. It's like test-driven development, but with docs!
+
+See `The Art of Documentation and Readme.md<https://pronovix.com/api-docs-london-2017#ben>`_ from API the Docs London for more. Or, if you're heading to `Write the Docs Prague<http://www.writethedocs.org/conf/eu/2017/>`_, check out one of the README-related sessions!
```
Comment (ericholscher) [inline docs/blog/newsletter-august-2017.rst]: Believe this link is formatted incorrectly.

File: docs/blog/newsletter-august-2017.rst [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+* Make the process for contributing as simple as possible
+
+As with all our stories, this conversation will continue to evolve on Slack, so if you have a favorite tip for or example of a maintainable docs system, definitely hop on Slack and share it!
+
+*************************
+Upcoming community events
+*************************
+
+**PRAGUE CONFERENCE COUNTDOWN: One Month Left!**
+We're likely to sell out in the next few weeks so `get your tickets ASAP <http://www.writethedocs.org/conf/eu>`_! And if you already have yours, don't forget to go `check out the full talk schedule <http://www.writethedocs.org/conf/eu/2017/news/announcing-schedule/>`_!
```
Comment (ericholscher) [inline docs/blog/newsletter-august-2017.rst]: This is a broken link (`http://www.writethedocs.org/conf/eu`)

File: docs/blog/newsletter-august-2017.rst (hunk 12 above)
Comment (ericholscher) [reply docs/blog/newsletter-august-2017.rst]: Every link in the section is, actually.

File: docs/blog/newsletter-august-2017.rst [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+For that last point, it can be hard to imagine what colors might look like to the various kinds of colorblindness. But this is the internet; there's tools for that! Folks suggested a couple of different options, including `Color Oracle <http://colororacle.org/>`_, a desktop simulator for the different types of colorblindness, and
+`Color Hexa <http://www.colorhexa.com/>`_, which provides a *wealth* of information about any color you look up, including how it appears for various types of colorblindness.
+
+************************
+Making docs maintainable
+************************
+This month the WTD hivemind tackled one of the holy grail questions: how do we establish a truly maintainable docs system? No surprise that we couldn't come up with a definitive set of practices, but folks contributed a wealth of great practical tips:
+* Implement well-structured information architecture
+* Automate as much as possible, especially in areas of rapid change, and make sure that automation scripts are easy to maintain
+*Examples: code includes, API reference docs, screenshots*
```
Comment (Bradamant3) [inline docs/blog/newsletter-august-2017.rst:55]: Is this the right formatting? Does it indent properly within the list? (Sorry, being lazy and not looking up here :-( )

Comment (kelly-o) [reply docs/blog/newsletter-august-2017.rst:55]: yep, seems to 👍

**Pull request merged**